libgraphics/src/context.cpp

```diff
@@ -2153,6 +2153,8 @@ static bool MCGContextApplyPaintSettingsToSkPaint(MCGContextRef self, MCGColor p
 			case kMCGImageFilterHigh:
                 r_paint . setFilterLevel(SkPaint::kHigh_FilterLevel);
 				break;
+            default:
+                break;
```

Comment:
@montegoulding Hmm, I'm worried that doing that puts the constant that describes the number of values in the enumeration a really long way from the definition of the enumeration.  I think it would probably be better to put the (non-`static`) definition of `kMCGImageFilterCount` as close as possible after the definition of the enumeration.  That will hopefully help make sure that if the `MCGImageFilter` enumeration is changed, the count gets changed too...